deno.jsonc

```diff
@@ -8,7 +8,8 @@
     "rules": {
       "exclude": [
         "no-explicit-any",
-        "camelcase"
+        "camelcase",
+        "no-import-prefix"
```

Comment:
I've run into linter issues with the existing code using inline imports. This rule [got added to the default ruleset in Deno 2.5](https://deno.com/blog/v2.5#no-import-prefix-lint-rule).

My assumption is the project's okay with inline imports, so I decided to exclude the rule instead. Let me know if I should rather refactor inline imports instead.